app/src/main/java/com/g11x/checklistapp/NavigationActivity.java

```diff
@@ -66,6 +67,15 @@
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
+    overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);
+    PreferredLanguageSupport.applyPreferredLanguage(this);
+    final Activity thisActivity = NavigationActivity.this;
```

Comment:
Is this an aesthetic thing? Or is there a technical reason for saving a final reference here? (this is the first time I've seen this pattern)